exasol_transformers_extension/deployment/utils.py

```diff
@@ -9,6 +15,10 @@
 
 DB_PASSWORD_ENVIRONMENT_VARIABLE = f"TE_DB_PASSWORD"
 BUCKETFS_PASSWORD_ENVIRONMENT_VARIABLE = f"TE_BUCKETFS_PASSWORD"
+SLC_PARTS_PREFIX_NAME = 'language_container_part_0'
+SLC_FINAL_NAME = "language_container.tar.gz"
+N_SLC_PARTS = 2
+GH_RELEASE_URL = "https://github.com/exasol/transformers-extension/releases/download"
```

Comment:
reading parameters also from `ENV` would enable the user(s) to have a `.env` file which can be put into `.gitingore` which would result in
* Less configuration within the code
* Less risk of leaking confidential settings (e.g. password)
* Less arguments to pass for the regular CLI invocations

Reply:
good idea, but there many configurable constants. it is better to handle in different ticket